release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -195,6 +195,12 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 [id="ocp-4-9-sandboxed-containers"]
 === {sandboxed-containers-first}
 
+FIPS mode is now automatically enabled for {sandboxed-containers-first}. {sandboxed-containers-first} deployed on an {product-title} cluster installed in FIPS mode will not taint the cluster's FIPS support. For more information, see
```

Comment:
```suggestion
FIPS mode is now automatically enabled for {sandboxed-containers-first}. {sandboxed-containers-first} that are deployed on an {product-title} cluster that is installed in FIPS mode will not taint the cluster's FIPS support. For more information, see
```

Reply:
Consider adding a big `TODO:` reminder for whichever links you want to end these items with.